packages/world/src/modules/keysintable/KeysInTableHook.sol

```diff
@@ -64,56 +53,20 @@ contract KeysInTableHook is IStoreHook {
         // Delete the list of keys in this table
         KeysInTable.deleteRecord(tableId);
       } else {
-        if (key.length > 0) {
-          bytes32[] memory lastKeyTuple = new bytes32[](key.length);
+        bytes32[] memory lastKeyTuple = new bytes32[](key.length);
 
-          bytes32 lastKey = KeysInTable.getItemKeys0(tableId, length - 1);
-          lastKeyTuple[0] = lastKey;
+        for (uint8 fieldIndex = 0; fieldIndex < key.length; fieldIndex++) {
```

Comment:
We should throw an explicit error if a the key has more than 5 elements, otherwise it would be very hard to debug if someone runs into it

Reply:
added a check during module installation, then the hook can assume it's correct and avoid using more gas